components/ILIAS/Dashboard/Administration/classes/class.ilDashboardSortationTableGUI.php

```diff
@@ -71,12 +73,13 @@ public function initData(): void
             $presentation_cb->setChecked($this->viewSettings->isViewEnabled($presentation_view));
             $presentation_cb->setValue('1');
             $presentation_cb->setDisabled(
-                $presentation_view === ilPDSelectedItemsBlockConstants::VIEW_RECOMMENDED_CONTENT
+                $disable ?: ($presentation_view === ilPDSelectedItemsBlockConstants::VIEW_RECOMMENDED_CONTENT)
```

Comment:
```suggestion
                $disable || ($presentation_view === ilPDSelectedItemsBlockConstants::VIEW_RECOMMENDED_CONTENT)
```

Just noticed this now :o

Reply:
I would still keep the `?:` due to the minor consideration that those 2 values are not logically connected but more functionally resulting in each other. E.g. would it be very unintuitive to flip the values which whould be a possibility the `||` would imply.

But as said thats more of a preference

Reply:
Or, for example, the `$presentation_view` logically is not required to be set, when a input is disabled anyways. This way i ensure this does not resolve into any conflict for such requests.

Or in other words: `?:` is more "strict" than `||`

Reply:
I disagree, `||` does not imply that you can switch them, because the later is never evaluated if the former is `true`, the same with `&&`: You cannot switch `$foo !== null && $foo->bar()` just as you cannot switch `$foo === null || $foo->bar()` around.

Reply:
Yes, im aware that those are not implicitly flippable.

What I tried to say is that most code assistent software provides optional suggestions for flipping those, which it does not for "shortened ifs".
This has its origin within the mathematically translating which differs between logical combinations (in which both combining parts are equal in their weight) and logical conclusions (in which 1 value results into another, but not obliged otherwise).

And since this is a logical conclusion ("Set Disable to XY, if the flag 'disabled' is not set") and no logical combination (Set disabled to the flag 'disabled' or to XY) i would prefer to use a conclusion operator here.

Reply:
I still disagree, as this is not the promise of `||` and `&&`, this would apply for `&` and `|` (**logical** `or` and `and`) but as this is just nit-picking, it's no really blocker